### ISSUE TITLE
Craft CMS 4 + Formie import form as update

### ISSUE THREAD

**Issue by sinnathurai** [verbb/formie#954]
**Description**
Hello, I just updated Craft CMS to 4.x and Formie to 2.0.0-beta.14. When I import a form with the "update" option. Then I get the following error message:

````
yii\base\UnknownMethodException: Calling unknown method: verbb\formie\fields\formfields\Group::getFields() in /var/www/html/vendor/yiisoft/yii2/base/Component.php:300
Stack trace:
#0 /var/www/html/vendor/verbb/formie/src/helpers/ImportExportHelper.php(146): yii\base\Component->__call('getFields', Array)
#1 /var/www/html/vendor/verbb/formie/src/controllers/ImportExportController.php(168): verbb\formie\helpers\ImportExportHelper::createFormFromImport(Array, Object(verbb\formie\elements\Form))
#2 [internal function]: verbb\formie\controllers\ImportExportController->actionImportComplete()
#3 /var/www/html/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#4 /var/www/html/vendor/yiisoft/yii2/base/Controller.php(178): yii\base\InlineAction->runWithParams(Array)
#5 /var/www/html/vendor/yiisoft/yii2/base/Module.php(552): yii\base\Controller->runAction('import-complete', Array)
#6 /var/www/html/vendor/craftcms/cms/src/web/Application.php(301): yii\base\Module->runAction('formie/import-e...', Array)
#7 /var/www/html/vendor/craftcms/cms/src/web/Application.php(625): craft\web\Application->runAction('formie/import-e...', Array)
#8 /var/www/html/vendor/craftcms/cms/src/web/Application.php(280): craft\web\Application->_processActionRequest(Object(craft\web\Request))
#9 /var/www/html/vendor/yiisoft/yii2/base/Application.php(384): craft\web\Application->handleRequest(Object(craft\web\Request))
#10 /var/www/html/web/index.php(30): yii\base\Application->run()
#11 {main}
````

Can you have a look?

Many thanks

**Steps to reproduce**
1. Main Navigation "Formie"
2. go to settings
3. go to Import/Export
4. select a file for upload
5. define as "update"

### Form settings
- Multi-page form: No
- Submission Method: -
- Client-side Validation: -
- Custom Form Templates: -

**Additional info**
- Plugin version: 2.0.0-beta.14
- Craft version: Craft CMS 4.0.5.1
- Multi-site: No

**Comment by engram-design** [comment]
Should be fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.0-beta.14",
  "...": "..."
}
```

Then run `composer update`.

**Comment by sinnathurai** [comment]
Many thanks, is fixed!

**Comment by engram-design** [comment]
Fixed in [2.0.0-beta.15](https://github.com/verbb/formie/releases/tag/2.0.0-beta.15)
